library/src/lib/table/table-body.component.ts

```diff
@@ -1,7 +1,9 @@
 import { Component } from '@angular/core';
 
 @Component({
-    selector: 'fd-table-body',
+    // TODO to be discussed
+    // tslint:disable-next-line:component-selector
+    selector: '[fd-table-body]',
     templateUrl: './table-body.component.html'
 })
 export class TableBodyComponent {}
```

Comment:
I think this will fix the issue good stuff

These components should be directives though, no? Files should be renamed as well in that case